windows/uninstall.lua

```diff
@@ -150,7 +147,11 @@ function remove_texmf_install()
     end
   end
   if install_was_present then
-    os.spawn("updmap")
+    --[[Since we removed some fonts, we need to rebuild the font databases.
+    Since this function is only used when extracting files from a TeXLive
+    texmf tree, we don't need to go through the distribution check.
+    ]]--
+    os.spawn("updmap-sys")
```

Comment:
`updmap` does nothing with gregorio fonts, it was just here to handle very old versions of Gregorio using `ovf`, `ofm`, etc. fonts, which were handled by `updmap`. For the new database, `luaotfload-tool` is what you're looking for